### PR TITLE
Enabling trash settings in HDFS

### PR DESCRIPTION
**Pull request by sjainit**
JIRA Ticket Number: LB-470 (https://tickets.metabrainz.org/projects/LB/issues/LB-470?filter=allopenissues)

### CODE REVIEW

File: config/core-site.xml [hunk 1]
```diff
@@ -4,4 +4,12 @@
         <name>fs.defaultFS</name>
         <value>hdfs://hadoop-master:9000/</value>
     </property>
+    <property>
+    	<name>fs.trash.interval</name>
+    	<value>3</value>
+	</property>
```
Comment (vansika) [inline config/core-site.xml:10]: The closing tag should be aligned with the opening tag.

File: config/core-site.xml [hunk 2]
```diff
@@ -4,4 +4,12 @@
...
+	<property>
```
Comment (vansika) [inline config/core-site.xml:11]: There is an extra tab here

File: config/core-site.xml [hunk 3]
```diff
@@ -4,4 +4,12 @@
...
+    	<name>fs.trash.checkpoint.interval</name>
+    	<value>1</value>
+	</property>
```
Comment (vansika) [inline config/core-site.xml:14]: Extra tab here too.

**Review by vansika — CHANGES_REQUESTED** [review]
Hey, Thanks for this PR. It looks amazing. Just a few indents and then it is good to go :)

**Comment by sjainit** [comment]
Thanks for the review @vansika. I have done the indentation changes.

**Review by vansika — APPROVED** [review]
Thanks :)

**Pull request merged**